### PR TITLE
Convert remaining package tests to transactions

### PR DESCRIPTION
**Pull request by shimonacarvalho**
### Summary 

All packages that were found to clone the database converted, except those that needed substantial changes. Those were written into separate tickets and added to the epic. 

### Testing

Running all server tests should suffice

For more about the changes, please see https://transcom.github.io/mymove-docs/docs/backend/testing/running-server-tests-inside-a-transaction/

### CODE REVIEW

File: pkg/awardqueue/awardqueue.go [hunk 1]
```diff
@@ -97,13 +97,6 @@ func (aq *AwardQueue) assignPerformanceBandsForTSPPerformanceGroup(appCtx appcon
 	return nil
 }
 
-// waitForLock MUST be called within a transaction!
-func waitForLock(appCtx appcontext.AppContext, id int) error {
-
-	// obtain transaction-level advisory-lock
-	return appCtx.DB().RawQuery("SELECT pg_advisory_xact_lock($1)", id).Exec()
-}
-
```
Comment (ahobson) [inline pkg/awardqueue/awardqueue.go:106]: I'm fine with this, but AFAICT, I don't think `AwardQueue` is even used anywhere any more. I wonder if we shouldn't remove it entirely?

File: pkg/migrate/migrate_test.go [hunk 2]
```diff
@@ -12,11 +12,12 @@ type MigrateSuite struct {
 	testingsuite.PopTestSuite
 }
 
+// This suite uses the high privileged role.
 func TestMigrateSuite(t *testing.T) {
 	ms := &MigrateSuite{
 		PopTestSuite: testingsuite.NewPopTestSuite(
 			"migrate",
-			testingsuite.WithHighPrivPSQLRole(),
+			testingsuite.WithHighPrivPSQLRole(), // WithPerTestTransaction cannot be used in conjunction with the HighPrivPSQLRole
```
Comment (ahobson) [inline pkg/migrate/migrate_test.go:20]: Nice!

**Review by ahobson — APPROVED** [review]

File: pkg/awardqueue/awardqueue.go (hunk 1 above)
Comment (shimonacarvalho) [reply pkg/awardqueue/awardqueue.go:106]: The whole package is unused? If so, I'll put in a ticket to remove entirely.

Comment (ahobson) [reply pkg/awardqueue/awardqueue.go:106]: I tried to search for usage of `AwardQueue` and I didn't find anything other than the award queue test

Comment (shimonacarvalho) [reply pkg/awardqueue/awardqueue.go:106]: Okay will create a separate pr to delete the whole pkg. Thanks!

**Pull request merged**